cmd/forwarder/http.go

```diff
@@ -131,6 +131,12 @@ func (s *httpServer) Run() error {
 			s.pAuthSuccesses.Inc(1)
 		}
 
+		var authUser string
+
```

Comment:
remove this extra line.

Reply:
I also recommend moving this declaration and if down to right above where it's used (line 155).